{"uuid": "281eb78a-cb32-4078-8ea2-09cf412d9861", "children": ["0a3bc987-b085-437e-a1a4-e34df7acb702"], "befores": [{"name": "create_order", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementClickInterceptedException: Message: Element <button class=\"Modal_modal__close_modified__3V5XS Modal_modal__close__TnseK\" type=\"button\"> is not clickable at point (1266,218) because another element <div class=\"Modal_modal_overlay__x2ZCr\"> obscures it; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#elementclickinterceptedexception\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nElementClickInterceptedError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:351:5\nwebdriverClickElement@chrome://remote/content/marionette/interaction.sys.mjs:175:11\ninteraction.clickElement@chrome://remote/content/marionette/interaction.sys.mjs:134:11\nclickElement@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:327:29\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:210:31\n\n", "trace": "  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1195, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 929, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\tests\\conftest.py\", line 88, in create_order\n    homepage.click_cross_button_order_confirm()\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 202, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\pages\\home_page.py\", line 42, in click_cross_button_order_confirm\n    self.click(self.locators.CROSS_BUTTON_ORDER_CONFIRM)\n    ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\pages\\base_page.py\", line 13, in click\n    self.wait.until(expected_conditions.element_to_be_clickable(element_locator)).click()\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 120, in click\n    self._execute(Command.CLICK_ELEMENT)\n    ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 573, in _execute\n    return self._parent.execute(command, params)\n           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 454, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Выбираем случайную булку", "status": "passed", "start": 1754739169890, "stop": 1754739169894}, {"name": "Выбираем случайный ингредиент", "status": "passed", "start": 1754739169894, "stop": 1754739169898}, {"name": "Перетаскиваем ингредиент в корзину", "status": "passed", "parameters": [{"name": "ingredient", "value": "<selenium.webdriver.remote.webelement.WebElement (session=\"acb10137-9867-4436-a492-8b18a41d14d2\", element=\"597fc2e0-7967-494a-955c-3b84175c101b\")>"}], "start": 1754739169898, "stop": 1754739169933}, {"name": "Перетаскиваем ингредиент в корзину", "status": "passed", "parameters": [{"name": "ingredient", "value": "<selenium.webdriver.remote.webelement.WebElement (session=\"acb10137-9867-4436-a492-8b18a41d14d2\", element=\"8f6dc103-1d4d-4b44-a6e1-1f8d0f1306b7\")>"}], "start": 1754739169933, "stop": 1754739169976}, {"name": "Нажимаем на кнопку Оформить заказ", "status": "passed", "start": 1754739169976, "stop": 1754739170072}, {"name": "Ждем появления высплавающего окна с подтверждением заказа", "status": "passed", "start": 1754739170072, "stop": 1754739170087}, {"name": "Ждем появления номера созданного заказа", "status": "passed", "start": 1754739170087, "stop": 1754739172731}, {"name": "Получаем номер созданного заказа", "status": "passed", "start": 1754739172731, "stop": 1754739172756}, {"name": "Нажимаем на крестик в всплывающем окне подтверждения заказа", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementClickInterceptedException: Message: Element <button class=\"Modal_modal__close_modified__3V5XS Modal_modal__close__TnseK\" type=\"button\"> is not clickable at point (1266,218) because another element <div class=\"Modal_modal_overlay__x2ZCr\"> obscures it; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#elementclickinterceptedexception\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nElementClickInterceptedError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:351:5\nwebdriverClickElement@chrome://remote/content/marionette/interaction.sys.mjs:175:11\ninteraction.clickElement@chrome://remote/content/marionette/interaction.sys.mjs:134:11\nclickElement@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:327:29\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:210:31\n\n", "trace": "  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 202, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\pages\\home_page.py\", line 42, in click_cross_button_order_confirm\n    self.click(self.locators.CROSS_BUTTON_ORDER_CONFIRM)\n    ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\pages\\base_page.py\", line 13, in click\n    self.wait.until(expected_conditions.element_to_be_clickable(element_locator)).click()\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 120, in click\n    self._execute(Command.CLICK_ELEMENT)\n    ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 573, in _execute\n    return self._parent.execute(command, params)\n           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 454, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"C:\\Users\\egorv\\PycharmProjects\\Diplom\\Diplom_3\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1754739172756, "stop": 1754739172787}], "start": 1754739169890, "stop": 1754739172796}], "start": 1754739169890, "stop": 1754739173015}